src/components/BookItem.js

```diff
@@ -0,0 +1,23 @@
+const books = // Initial state:
```

Comment:
[OPTIONAL]
- [ ] As required by the project requirements, the initial state of the arrays of books should be empty.

<img width="435" alt="image" src="https://github.com/olanikegloria/react-bookstore/assets/52934128/9f451bd8-d6c0-417e-80f1-387dc6ab62e9">